cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -26,93 +32,48 @@
 {% block hero -%}
     {% if prototype %}
         {% include 'molecules/hero.html' %}
-    {% endif %}
-    {% for block in page.hero %}
-        {{ render_stream_child(block) }}
-    {% endfor %}
+    {% else %}
+        {% for block in page.header -%}
+            {% if 'hero' in block.block_type %}
+                {{ render_stream_child(block) }}
+            {% endif %}
+        {%- endfor %}
+    {%- endif %}
 {% endblock %}
 
-{% set lipsumText = lipsum(1, false, 10, 25) | safe %}
 {% block content_main %}
-    {% if page.introduction or prototype %}
-        <div class="block
-                 block__flush-top">
-            {% if prototype %}
-                {% include 'molecules/text-introduction.html' %}
-            {% endif %}
-            {% for block in page.introduction %}
-                {{ render_stream_child(block) }}
-            {% endfor %}
-        </div>
-    {% endif %}
+    {% if prototype %}
+        {% include 'molecules/text-introduction.html' %}
```

Comment:
I'd re-add all the `block` wrappers that were removed to fix the layout, then we can address whether they should be in the templates or the molecules/organisms in another PR.